### ISSUE TITLE
ERROR Fetching registry.npmjs.org/*/* failed!

### ISSUE THREAD

**Issue by ExE-Boss** [pnpm/pnpm#1269]
### pnpm version:
`2.10.5`, `2.11.1`

### Code to reproduce the issue:
`pnpm install`

https://gist.github.com/ExE-Boss/b5a898627f1d9ef5ddf5393416e239c7#file-pnpm-debug-log

### Expected behavior:
`pnpm` installs the dependencies successfully

### Actual behavior:
`pnpm` fails to fetch from [`registry.npmjs.org`](registry.npmjs.org) for whatever reason

### Additional information:
 - `node -v` prints: `v10.6.0`
 - Windows, OS X, or Linux?: Windows

**Comment by zkochan** [comment]
What does this return `npm get fetch-retries`?

Does this error happen all the time or from time-to-time? Do you know on which tarball it failed? (Seems like we have to improve logging inside [@pnpm/tarball-fetcher](https://github.com/pnpm/tarball-fetcher) because it did not report the tarball that failed)

**Comment by ExE-Boss** [comment]
It started happening around the time I opened the issue.

**Comment by ExE-Boss** [comment]
@zkochan I just tested `npm get fetch-retries`, and it returns `2`, also this still happens.

**Comment by ExE-Boss** [comment]
Turns out, deleting the packages from `~/.pnpm-store` that failed fetching fixed it for me (the directories must have somehow gotten corrupted, as Windows Explorer was also unable to read them for whatever reason).

**Comment by zkochan** [comment]
1. More informative error message
2. Maybe it happened because the cached tarball was corrupted. pnpm has to download it from the registry in that case

**Comment by ExE-Boss** [comment]
It wasn’t just the tarball, I couldn’t even enter the extracted directory with Windows Explorer providing a cryptic error message when I tried to do so:
```ini
[Window Title]
Location is not available

[Content]
C:\Users\ExE Boss\.pnpm-store\2\registry.npmjs.org\boxen\0.5.1\package is unavailable.
If the location is on this PC, make sure that the device or drive is connected
or the disc is inserted, then try again. If the location is on a network, make sure that
you're connected to the network or Internet, then try again. If the location still can't
be found, it might have been moved or deleted.

[OK]
```

**Comment by zkochan** [comment]
I believe this was fixed in scope of #1476
